src/methods/broadcast_tx_commit.rs

```diff
@@ -1,3 +1,76 @@
+//! Sends blocking transactions.
+//!
+//! Sends a signed transaction to the RPC and waits until transaction is fully complete.
+//!
+//! ## Example
+//!
+//! ```
```

Comment:
> This is quite long. Plus, we can't realistically test submitting transactions on-chain here. Let's strip this to showing just how to construct a request, without sending it to the client. By doing that, we can clean up a lot of boilerplate.

_Originally posted by @miraclx in https://github.com/near/near-jsonrpc-client-rs/pull/79#discussion_r856859313_